mmv1/products/privateca/CertificateTemplate.yaml

```diff
@@ -12,14 +12,36 @@
 # limitations under the License.
 
 --- !ruby/object:Api::Resource
-name: 'CertificateTemplate'
+name: CertificateTemplate
 base_url: projects/{{project}}/locations/{{location}}/certificateTemplates
+create_url: projects/{{project}}/locations/{{location}}/certificateTemplates?certificateTemplateId={{name}}
 self_link: projects/{{project}}/locations/{{location}}/certificateTemplates/{{name}}
-# This resource is only used to generate IAM resources. They do not correspond to real
-# GCP resources, and should not be used to generate anything other than IAM support.
-exclude_resource: true
+update_verb: :PATCH
+update_mask: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/certificate-authority-service'
+  api: 'https://cloud.google.com/certificate-authority-service/docs/reference/rest'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+    path: 'name'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
 description: |
-  Only used to generate IAM resources
+  A CertificateTemplate refers to a managed template for certificate issuance.
```

Comment:
Looks like this originally had a more in-depth description, can you add that back?
https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10527-old..auto-pr-10527#diff-8fc7d1e5f797d25b4b2bdb42ae8679906db4a7b151a3ec25adf939ef65aaa098L18